src/.schema.sql

```diff
@@ -13,3 +13,6 @@ CREATE SCHEMA kubercade AUTHORIZATION nrhwqomw
     message     varchar(400) NOT NULL,
     datetime    varchar(40) NOT NULL
   );
+
+jxzxrgmv
+nrhwqomw
```

Comment:
What are these doing here?

Reply:
magic strings for AUTHRORIZATION is a bit odd

Reply:
Oops it seems like I forgot to delete those, fixed in 9fa2971